contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -1,7 +1,7 @@
 #
 # Cilium runtime base image
 #
-FROM docker.io/library/ubuntu:18.04 as runtime-base
+FROM docker.io/library/ubuntu:20.04 as runtime-base
```

Comment:
Was this necessary to implement the other suggested changes?

I think @aanm had some thoughts on what criteria we may need to move to a new base image (including eg other potential pitfalls).

Reply:
Yes, to have 1.8 version of iptables.
The default iptables version for 18.04 is 1.6. Not sure is there a way to install newer version in 18.04.
Is there some certification process for the image change?

Reply:
Ok, in that case we should also bump the base image of the `./Dockerfile.builder`

Reply:
Done.